WordPress/src/main/java/org/wordpress/android/util/experiments/SiteNameABExperiment.kt

```diff
@@ -0,0 +1,6 @@
+package org.wordpress.android.util.experiments
+
+import javax.inject.Inject
+
+class SiteNameABExperiment
+@Inject constructor(exPlat: ExPlat) : Experiment(name = "wpandroid_site_name_experiment", exPlat)
```

Comment:
Created an experiment named `wpandroid_site_name_v1` before I got a look at the name here :(

Reply:
Updated with 8f361d4672f53abcd96c4f02d82c23d2b59c7d28

Reply:
Thanks for updating (and also fixing the ktlint :+1: )